### PR TITLE
qm-disk-import: add page; qm-importdisk: add alias page

### PR DESCRIPTION
**Pull request by BabaYaga1221**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by BabaYaga1221** [comment]
@kbdharun @acuteenvy Would you like to suggest any change? Thanks

File: pages/linux/qm-importdisk.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# qm importdisk
+
+> Import disk image to a VM as unused disk.
+> The supported image formats for qemu-img, such as raw, qcow2, qed, vdi, vmdk, and vhd, must be used.
```
Comment (kbdharun) [inline pages/linux/qm-importdisk.md:4]:
```suggestion
> The supported image formats for `qemu-img`, such as raw, qcow2, qed, vdi, vmdk, and vhd, must be used.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/qm-disk-import.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# qm disk import
+
+> Import disk image to a VM as unused disk.
```
Comment (acuteenvy) [inline pages/linux/qm-disk-import.md:3]:
```suggestion
> Import a disk image to a virtual machine as an unused disk.
```

**Review by acuteenvy — APPROVED** [review]

**Comment by acuteenvy** [comment]
@BabaYaga1221 you can apply suggestions directly on GitHub, no need to edit the files again yourself.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**